server/text_generation_server/models/flash_llama.py

```diff
@@ -34,13 +34,22 @@ def __init__(
         else:
             raise NotImplementedError("FlashLlama is only available on GPU")
 
-        tokenizer = LlamaTokenizer.from_pretrained(
-            model_id,
-            revision=revision,
-            padding_side="left",
-            truncation_side="left",
-            trust_remote_code=trust_remote_code,
-        )
+        try:
+            tokenizer = LlamaTokenizer.from_pretrained(
+                model_id,
+                revision=revision,
+                padding_side="left",
+                truncation_side="left",
+                trust_remote_code=trust_remote_code,
+            )
+        except Exception:
```

Comment:
Can this exception be more specific?
Also, where is this new try/catch coming from?

Reply:
For the quantized model I save the output of `AutoTokenizer.from_pretrained(..).save_pretrained()` which kills the slow tokenizer.

For the exception, I'm not sure what kind it is.

We could also just `AutoTokenizer` which would drop the exception and just work.